orchestrator-salt/src/main/resources/salt/salt/log4jpatch/init.sls

```diff
@@ -0,0 +1,10 @@
+{% if salt['pillar.get']('log4j_patch_enabled',False) == True %}
+run_log4j_patch:
```

Comment:
I think this is invoked for every highstate

Reply:
yes, that was our plan, wasn't it?

Reply:
otherwise I think we need a precise declaration about when and how should we apply this state, I can restrict this state with a grains role, or we can place a file, as @lacikaaa suggested on the nodes, where the script were already applied and based on that file we will not execute it once again